cafebabel/static/js/script.js

```diff
@@ -113,3 +113,25 @@ Array.from(document.querySelectorAll('h1.edit input')).forEach(inputh1 => {
     () => inputh1.value || h1.classList.remove('active')
   )
 })
+
+const tagLinks = document.querySelectorAll('.tags-list a') /* https://stackoverflow.com/a/46791132/6481285 */
```

Comment:
Proposal (untested):

```javascript
Array.from(document.querySelectorAll('.tags-list a')).forEach(tag => {
  const content = tag.innerHTML.trim().toLowerCase()
  if (content in ['impact', 'experience', 'raw', 'creative'])
    tag.classList.add(content)
})
```

Reply:
it doesn't work :-(

Reply:
https://leantesting.com/write-good-bug-report/ 👼

Reply:
it doesn't add any class. Console doesn't display any error. I can't get anything when I console `tag.innerHTML.trim().toLowerCase()`. I don't know what else to report to be more helpful :/

PS : never trust any website where they place a 200x200px-cover picture in a 600px-wide container 👼